server/public/model/client4.go

```diff
@@ -8305,50 +8297,6 @@ func (c *Client4) GetMyIP(ctx context.Context) (*GetIPAddressResponse, *Response
 	return response, BuildResponse(r), nil
 }
 
-func (c *Client4) CreateCustomerPayment(ctx context.Context) (*StripeSetupIntent, *Response, error) {
```

Comment:
Not that I expect that these to be used by anyone else, but removing these methods is technically a breaking change. I'm 0/5 though because I know I tend to be overly cautious about breaking this interfaces.

Worst case, we could always stub them out instead of calling the APIs which no longer exist